packages/menu/src/MenuContainer.tsx

```diff
@@ -29,5 +29,12 @@ MenuContainer.propTypes = {
   defaultExpanded: PropTypes.bool,
   selectedItems: PropTypes.arrayOf(PropTypes.any),
   focusedValue: PropTypes.oneOfType([PropTypes.string]),
-  defaultFocusedValue: PropTypes.oneOfType([PropTypes.string])
+  defaultFocusedValue: PropTypes.oneOfType([PropTypes.string]),
+  restoreFocus: PropTypes.bool
+};
+
+MenuContainer.defaultProps = {
+  defaultExpanded: false,
+  restoreFocus: true,
+  rtl: false
```

Comment:
IIRC it would be acceptable to leave `defaultExpanded` and `rtl` as undefined since they should be read as falsey anyway.

Reply:
+1 – Garden doesn't/shouldn't specify `false` default props